docs/analyzers/TypeAnnotateStringFunction.md

```diff
@@ -0,0 +1,26 @@
+---
+title: TypeAnnotateStringFunctionAnalyzer
+category: analyzers
+categoryindex: 1
+index: 6
+---
+
+# TypeAnnotateStringFunction
+
+## Problem
+
+Using the `string` function can catch you off guard when refactoring types.
+
+```fsharp
+let v = 1 // Changing this value won't affect `s`
```

Comment:
To make it a bit more clear, how about: Changing the type of this value won't affect `s`